batch7crud-roll1/src/main/java/com/lftechnology/batch7crud/dao/EmployeeDAOImpl.java

```diff
@@ -0,0 +1,115 @@
+package com.lftechnology.batch7crud.dao;
+
+import com.lftechnology.batch7crud.database.DatabaseConnection;
+import com.lftechnology.batch7crud.exception.DataException;
+import com.lftechnology.batch7crud.model.Employee;
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class EmployeeDAOImpl{
+
+	DatabaseConnection databaseConnection = new DatabaseConnection();
+
+	public int insert(Employee employee) throws DataException {
+
+        try {
+            Connection connection = databaseConnection.getConnection();
+            String sqlQuery = "insert into employee(id,username,password,fullname,department,address) values(?,?,?,?,?,?)";
+            PreparedStatement statement = connection.prepareStatement(sqlQuery);
+            statement.setInt(1, employee.getId());
+            statement.setString(2, employee.getUserName());
+            statement.setString(3, employee.getPassword());
+            statement.setString(4, employee.getFullName());
+            statement.setString(5, employee.getDepartment());
+            statement.setString(6, employee.getAddress());
+            return statement.executeUpdate();
+        }catch (SQLException e){
+            throw new DataException();
+        }
+	}
+	
+	public List<Employee> fetch(int page) throws DataException{
+
+        try{
+            Connection connection = databaseConnection.getConnection();
+            String sqlQuery = "select * from employee order by id limit 2";
+            PreparedStatement statement = connection.prepareStatement(sqlQuery);
+            ResultSet result = statement.executeQuery();
+            List<Employee> employees = new ArrayList<Employee>();
+            Employee employee = null;
+
+            while(result.next()){
+                employee = new Employee();
+                employee.setId(result.getInt("id"));
+                employee.setUserName(result.getString("userName"));
+                employee.setPassword(result.getString("password"));
+                employee.setFullName(result.getString("fullName"));
+                employee.setAddress(result.getString("address"));
+                employee.setDepartment(result.getString("department"));
+                employees.add(employee);
+            }
+            return employees;
+        }catch (SQLException e){
+            throw new DataException();
+        }
+
+	}
+
+    public Employee getEmployeeById(int id) throws DataException{
```

Comment:
"get" is not preferred because it feels like a getter method.
use fetchById(int employeeId) - it's more descriptive.